fuse/data/ops/ops_common.py

```diff
@@ -351,6 +353,46 @@ def __call__(self, sample_dict: NDict, op_id: Optional[str], keep_keypaths:List[
         sample_dict = NDict()
         for k in keep_keypaths:
             sample_dict[k] = prev_sample_dict[k]
+
+        return sample_dict
+
+
+class OpMap(OpBase):
+    """
+    Mapping values. The mapping specified in a dictionary
+    """
+    def __init__(self, map: dict):
+        super().__init__()
+        self._map = map
+
+    def __call__(self, sample_dict: NDict, key_in: str, key_out: str) -> Union[None, dict, List[dict]]:
+        value = sample_dict[key_in]
+        sample_dict[key_out] = self._map[value]
+
         return sample_dict
 
 
+class OpToOneHot(OpBase):
+    """
+    Map category value to one hot vector
+    """
+    def __init__(self, num_classes: int):
+        super().__init__()
+        self._num_classes = num_classes
+        
+    def __call__(self, sample_dict: NDict, key_in: str, key_out: str) -> Union[None, dict, List[dict]]:
+        value = sample_dict[key_in]
+        one_hot = np.zeros(self._num_classes)
+        one_hot[value] = 1.0
+        sample_dict[key_out] = one_hot
+
+        return sample_dict
+
+
+class OpConcat(OpBase):
```

Comment:
can you please add a docstring and also an example of input and output for each Op class ?